app/components/DosesDescriptionList/DosesDescriptionList.tsx

```diff
@@ -59,7 +61,8 @@ export const DosesDescriptionList = ({
       dosesPercent={dhbData.secondDosesPercentage}
     >
       <dd className={styles["vaccine-count"]}>
-        {numberFormatter.format(dhbData.secondDosesTo90Percent)} left
+        {numberFormatter.format(dhbData.secondDosesTo90Percent)} left{" "}
+        {!dhbData.hasMetSecondDoseTarget && `(${dhbData.secondDosesChange})`}
```

Comment:
Minor: Preference to read in the positive intent

```suggestion
        {dhbData.hasMetSecondDoseTarget ? null : `(${dhbData.secondDosesChange})`}
```

```suggestion
        {dhbData.hasMetSecondDoseTarget || `(${dhbData.secondDosesChange})`}
```

Reply:
Removed from UI for now